deepmd/utils/argcheck.py

```diff
@@ -2280,6 +2321,36 @@ def loss_dos():
     ]
 
 
+@loss_args_plugin.register("property")
+def loss_property():
+    doc_loss_func = "The loss function to minimize, such as 'mae','smooth_mae'."
+    doc_metric = "The metric such as 'mae','smooth_mae' for display."
```

Comment:
Please list all the options here.

Reply:
I have changed it to:
"The metric for display. This list can include 'smooth_mae', 'mae', 'mse' and 'rmse'."